apps/nextjs-website/src/components/organisms/ChatbotHistoryLayout/ChatbotHistoryLayout.tsx

```diff
@@ -0,0 +1,31 @@
+import ChatbotHistoryList from '@/components/molecules/ChatbotHistoryList/ChatbotHistoryList';
+import { PaginatedSessions } from '@/lib/chatbot/queries';
+import { Pagination, Stack } from '@mui/material';
+import { useEffect } from 'react';
+
+type ChatbotHistoryLayoutProps = {
+  paginatedSessions: PaginatedSessions;
+  getSessions: (page: number) => null;
```

Comment:
maybe this function could be call `getSessionByPage` to be more explicit

Reply:
done